### ISSUE TITLE
mbp m1 启动报错

### ISSUE THREAD

**Issue by ytlrr0** [nacos-group/nacos-docker#199]
2.0.2版本
本机器环境: macbookpro m1 

 ERROR org.springframework.boot.SpringApplication - Application run failed

com.alibaba.nacos.api.exception.runtime.NacosRuntimeException: ErrCode:500, ErrMsg:Function not implemented

at com.alibaba.nacos.core.listener.StartingApplicationListener.loadPreProperties(StartingApplicationListener.java:160)

at com.alibaba.nacos.core.listener.StartingApplicationListener.environmentPrepared(StartingApplicationListener.java:99)

at com.alibaba.nacos.core.code.SpringApplicationRunListener.environmentPrepared(SpringApplicationRunListener.java:65)

at org.springframework.boot.SpringApplicationRunListeners.environmentPrepared(SpringApplicationRunListeners.java:53)

at org.springframework.boot.SpringApplication.prepareEnvironment(SpringApplication.java:342)

at org.springframework.boot.SpringApplication.run(SpringApplication.java:305)

at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)

at org.springframework.boot.SpringApplication.run(SpringApplication.java:1204)

at com.alibaba.nacos.Nacos.main(Nacos.java:35)

at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

at java.lang.reflect.Method.invoke(Method.java:498)

at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)

at org.springframework.boot.loader.Launcher.launch(Launcher.java:108)

at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)

at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:467)

Caused by: com.alibaba.nacos.api.exception.NacosException: java.io.IOException: Function not implemented

at com.alibaba.nacos.sys.file.WatchFileCenter$WatchDirJob.<init>(WatchFileCenter.java:189)

at com.alibaba.nacos.sys.file.WatchFileCenter.registerWatcher(WatchFileCenter.java:97)

at com.alibaba.nacos.core.listener.StartingApplicationListener.registerWatcher(StartingApplicationListener.java:166)

at com.alibaba.nacos.core.listener.StartingApplicationListener.loadPreProperties(StartingApplicationListener.java:158)

... 16 common frames omitted

Caused by: java.io.IOException: Function not implemented

at sun.nio.fs.LinuxWatchService.<init>(LinuxWatchService.java:64)

at sun.nio.fs.LinuxFileSystem.newWatchService(LinuxFileSystem.java:47)

at com.alibaba.nacos.sys.file.WatchFileCenter$WatchDirJob.<init>(WatchFileCenter.java:184)

... 19 common frames omitted

**Comment by paderlol** [comment]
这个应该跟基础镜像没有支持ARM架构有关系

**Comment by Azleal** [comment]
遇到了同样的问题

**Comment by liyiwei979621500** [comment]
遇到了同样的问题

**Comment by WillemJiang** [comment]
遇到了同样问题！

**Comment by SAMZONG** [comment]
遇到了同样的问题

**Comment by SAMZONG** [comment]
https://github.com/alibaba/nacos/issues/6340

@WillemJiang @ytlrr0 @Azleal @paderlol 👆🏻 issue 给出了解决办法，使用调整后的 nacos images

```yaml
services:
  nacos:
    image: zhusaidong/nacos-server-m1:2.0.3  # 使用这个镜像
    container_name: nacos-standalone
    environment:
    - PREFER_HOST_MODE=hostname
    - MODE=standalone
    volumes:
    - ./standalone-logs/:/home/nacos/logs
    - ./init.d/custom.properties:/home/nacos/init.d/custom.properties
```

**Comment by paderlol** [comment]
Fixed #257
